### PR TITLE
Updated docs

### PR DESCRIPTION
**Pull request by mayeedwin**
- All to ES6 Syntax
- Changed **T.** to semantic **twit**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,10 +12,10 @@ npm install twit
 
 ## Usage:
 
-```javascript
-var Twit = require('twit')
+```js
+const twit = require('twit')
```
Comment (ttezel) [inline README.md]: should be `const Twit` here

File: README.md [hunk 2]
```diff
@@ -66,43 +66,43 @@ T.get('account/verify_credentials', { skip_status: true })
...
-T.get('users/suggestions/:slug', { slug: 'funny' }, function (err, data, response) {
+twit.get('users/suggestions/:slug', { slug: 'funny' }, (err, data, response) => {
   console.log(data)
 })
 
 //
 // post a tweet with media
 //
-var b64content = fs.readFileSync('/path/to/img', { encoding: 'base64' })
+let b64contentwit = fs.readFileSync('/path/to/img', { encoding: 'base64' })
```
Comment (ttezel) [inline README.md]: this `let` and all of the `let`s below can be changed to `const`, since we don't edit these variables

**Review by ttezel — CHANGES_REQUESTED** [review]
Thank you so much @mayeedwin! I really appreciate you cleaning up the docs. Just a couple comments inline

File: README.md (hunk 2 above)
Comment (mayeedwin) [reply README.md]: Great, on it.

**Comment by mayeedwin** [comment]
Done!

File: README.md [hunk 3]
```diff
@@ -66,43 +66,43 @@ T.get('account/verify_credentials', { skip_status: true })
...
-T.get('users/suggestions/:slug', { slug: 'funny' }, function (err, data, response) {
+twit.get('users/suggestions/:slug', { slug: 'funny' }, (err, data, response) => {
   console.log(data)
 })
 
 //
 // post a tweet with media
 //
-var b64content = fs.readFileSync('/path/to/img', { encoding: 'base64' })
+const b64contentwit = fs.readFileSync('/path/to/img', { encoding: 'base64' })
```
Comment (ttezel) [inline README.md]: small typo: this should be `b64contentwit`

File: README.md [hunk 4]
```diff
@@ -66,43 +66,43 @@ T.get('account/verify_credentials', { skip_status: true })
...
 // first we must post the media to Twitter
-T.post('media/upload', { media_data: b64content }, function (err, data, response) {
+twit.post('media/upload', { media_data: b64content }, (err, data, response) => {
   // now we can assign alt text to the media, for use by screen readers and
   // other text-based presentations and interpreters
-  var mediaIdStr = data.media_id_string
-  var altText = "Small flowers in a planter on a sunny balcony, blossoming."
-  var meta_params = { media_id: mediaIdStr, alt_text: { text: altText } }
+  const mediaIdStr = data.media_id_string
+  const altTextwit = "Small flowers in a planter on a sunny balcony, blossoming."
```
Comment (ttezel) [inline README.md]: small typo here: should be `altText`

File: README.md [hunk 5]
```diff
@@ -281,18 +285,18 @@ stream.on('message', function (msg) {
...
-```javascript
-stream.on('tweet', function (tweet) {
+```js
+stream.on('tweet', tweet => {
   //...
 })
 ```
 
-## event: 'delete'
+## event: 'deconste'
```
Comment (ttezel) [inline README.md]: small typo: should be `delete` here and below

File: README.md [hunk 6]
```diff
@@ -592,7 +596,7 @@ THE SOFTWARE.
   * Use JSON payload in request body for new DM endpoints.
 
 ### 2.2.8
-  * Add support for HTTP DELETE; you can now `T.delete(...)`.
+  * Add support for HTTP DEconstE; you can now `twit.deconste(...)`.
```
Comment (ttezel) [inline README.md]: typo here too, should be DELETE and twit.delete

**Review by ttezel — CHANGES_REQUESTED** [review]
Thanks @mayeedwin! Few small typos to fix, then it'll be good to go!

**Comment by mayeedwin** [comment]
Oh, yeah. Oh my. I did replace all for  `let`, on it!

File: README.md (hunk 3 above)
Comment (mayeedwin) [reply README.md]: Actually should be, `b64content`, I was to change  `var` to `const` only

Comment (ttezel) [reply README.md]: ah yes, sorry I meant to type `b64content`, this one should be changed to `b64content` like you mentioned :)

File: README.md (hunk 4 above)
Comment (ttezel) [reply README.md]: this one still needs to be changed to `altText` :)

File: README.md (hunk 5 above)
Comment (ttezel) [reply README.md]: this one still needs to be changed back to `delete`, along with the ones below :)

File: README.md (hunk 6 above)
Comment (ttezel) [reply README.md]: this one still needs updating :)

**Review by ttezel — CHANGES_REQUESTED** [review]
Thanks @mayeedwin! Just a few left to be resolved, I think you marked some of the edits resolved but they still need some typo fixes :)

**Comment by mayeedwin** [comment]
> Thanks @mayeedwin! Just a few left to be resolved, I think you marked some of the edits resolved but they still need some typo fixes :)

Oh, yeah. Noticed that I had not pushed the changes. Should be okay now. Thanks for your patience. Feel free to help fix any other issue.

**Pull request closed without merging**